### DISCUSSION TITLE
Download certificate crt and key files

### DISCUSSION THREAD

**Discussion by simon-appleyard** [ravendb/ravendb#14637] [General]
Hello,

We have updated our server certificate and now need to download the the .crt and .key files again.

https://ravendb.net/docs/article-page/5.4/csharp/client-api/rest-api/rest-api-intro
says its possible to use <the URL of your server>/admin/debug/cluster-info-package to download the zip file.

However, the downloaded zip file does not contain the certificate files.

Is there another way to download them.

Many Thanks

**Comment by garayx** [comment]
That is a typo in the documentation, we will fix it,

How did you update the server certificate ? Did you receive a pfx file ? I think you can convert it to a .crt and .key files.

**Comment by ayende** [comment]
Note that the cluster info package will _not_ contain the private key of the cluster certificate.
If you have the `pfx` file, you can just convert that to the `crt` / `key` files.
However, if you are using the cluster certificate as the auth mechanism to the server, please generate a proper client certificate, instead.

**Comment by simon-appleyard** [comment]
Thanks. 
We updated the server certificate by replacing the file in the Server directory for each node.
Yes, we have managed to create crt/key files from the pfx.
We have made sure to use a client certificate for authorisation to the server.
A developer is still having problems using the crt/key files in a curl command but that could be related to other issues.
Thanks
